LayoutTests/platform/mac/TestExpectations

```diff
@@ -2530,8 +2530,6 @@ webkit.org/b/264177 requestidlecallback/requestidlecallback-deadline-shortened-b
 
 webkit.org/b/264527 imported/w3c/web-platform-tests/css/css-cascade/idlharness.html [ Pass Failure ]
 
-webkit.org/b/264607 [ Sonoma+ ] css1/box_properties/acid_test.html [ Pass Timeout ]
-
```

Comment:
note that neither that bug (https://bugs.webkit.org/show_bug.cgi?id=264607) nor the results database (https://results.webkit.org/?suite=layout-tests&suite=layout-tests&suite=layout-tests&test=css1%2Fbox_properties%2Facid_test.html&test=css2.1%2Ft09-c5526c-display-00-e.html&test=fast%2Fblock%2Fbasic%2F011.html&platform=mac) show any pass/timeout flakiness—there's some FAIL a while ago, but that was resolved by rebaselining in https://github.com/WebKit/WebKit/commit/8849660df1a921225fff49a437e79b9c7fd04ebd, thus let's just remove this line entirely

Reply:
Reasonable but if the test is being removed in its entirely then we should do `find-expectations <test name>` to remove all expectations for each test we remove. (acid_test, and 011)

Reply:
Yeah, I'd checked there weren't any other references. (check-webkit-style had caught the above!)